### PR TITLE
Adds magazines to loadouts

### PR DESCRIPTION
**Pull request by Al-1ce**
# About the pull request

This pull request adds all of the various Boots! editions to the loadout selection. I threw in a crayon and a hubba-hubba mag because I felt like those were missing as well.

# Explain why it's good for the game

* Boots! is _THE_ USCM magazine. You'd expect there to be at least a dozen fanboys on the Almayer carrying an edition or two. 

* Crayons are sovlful. What else can I say? (I added the rainbow one instead of a dozen different colors for ease)

* Hubba hubba.

# Testing Photographs and Procedure
Irrelevant.

# Changelog

:cl:
add: Adds various magazines and a crayon to loadout selection.
/:cl:

### CODE REVIEW

File: code/modules/client/preferences_gear.dm [hunk 1]
```diff
@@ -415,6 +415,33 @@ var/global/list/gear_datums_by_name = list()
 	display_name = "Camera"
 	path = /obj/item/device/camera
 
+/datum/gear/toy/mags
+	cost = 1
+
+/datum/gear/toy/mags/magazinedirty
```
Comment (morrowwolf) [inline code/modules/client/preferences_gear.dm:421]: snake case all these type paths

snake_case_all_these_type_paths <--- example

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/client/preferences_gear.dm (hunk 1 above)
Comment (Al-1ce) [reply code/modules/client/preferences_gear.dm:421]: I'm not sure what you're asking here. The other typepaths aren't snake-cased, what causes these ones the need to be?

Do you want me to change `/datum/gear/toy/mags/magazinedirty/` into `/datum_gear_toy_mags_magazinedirty/` or something? The example you gave is a little vague.

Comment (morrowwolf) [reply code/modules/client/preferences_gear.dm:421]: /datum/gear/toy/mags/magazine_dirty

Comment (Al-1ce) [reply code/modules/client/preferences_gear.dm]: Should be fine now?

Comment (morrowwolf) [reply code/modules/client/preferences_gear.dm]: Beautiful!

**Review by harryob — APPROVED** [review]

**Pull request merged**